components/LessonTitleCard.test.js

```diff
@@ -25,3 +51,23 @@ describe('LessonTitleCard component', () => {
     expect(container).toMatchSnapshot()
   })
 })
+
+describe('LessonTitleCard component on Review Page', () => {
+  const props = {
+    lessonCoverUrl: 'coverUrl',
+    lessonUrl: 'lessonUrl',
+    lessonTitle: 'Test Lesson',
+    lessonId: '0',
+    isPassed: true
+  }
+
+  test('Should render default layout for all screens', async () => {
```

Comment:
Unfortunately, snapshot testing fails short when it comes for media query based rules. You can check `LessonTitleCard.test.js.snap` file:
`LessonTitleCard component on Review Page Should render default layout for all screens 2` and `LessonTitleCard component on Review Page Should render default layout for all screens 2` snapshots are the same. 
Jest compares only classnames and they don't change.

Reply:
I haven't changed this, this is how tests were before. They are just testing whether `show challenges` is shown on mobile or not which snapshot can detect. I made two describe blocks for review and curriculum page, tests that differ for the two pages can be put in these. But for now, on curriculum page `Show challenge` should be shown and on review page no change should be made on mobile (it is handled by bootstrap's responsive css)
So since this describe is checking that the markup remains unchanged regardless of window width, the snapshots should be same right ?

Reply:
You're right, layouts should be the same in this case. But in general it's better to avoid relying on snapshots when you need to test different layouts.